homework-submission/barbarosapp/src/components/item.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import CheckBox from "./checkBox";
+import RemoveButton from "./removeButton";
+
+class Item extends React.Component {
+  render() {
+    const { name, deadLine, handleCheck, handleRemove, done, id } = this.props;
+
```

Comment:
I will complete the review after this hint. 
```suggestion
console.log(deadLine)
```

if you get stuck and you want me to continue let me know 😬

Reply:
Corrected. Thanks for help